cli/src/commands/sidekick.rs

```diff
@@ -63,40 +59,30 @@ impl SubCommand {
 		};
 
 		// Prepare the link
-		let prepare_res = rivet_api::apis::cloud_devices_links_api::cloud_devices_links_prepare(
-			&openapi_config_cloud_unauthed,
-		)
-		.await;
-		if let Err(err) = prepare_res.as_ref() {
-			struct_fmt::print(
-				&Format::Json,
-				&SideKickResponse::Err(json!({
-					"error": err.to_string(),
-				})),
-			)?;
-			bail!("Error: {err:?}");
-		}
-		let prepare_res = prepare_res.context("cloud_devices_links_prepare")?;
-
-		struct_fmt::print(
-			&Format::Json,
-			&SideKickResponse::Ok(json!({
-				"device_link_url": prepare_res.device_link_url,
-				"device_link_token": prepare_res.device_link_token,
-			})),
-		)?;
+		let prepare_res = unwrap!(
+			apis::cloud_devices_links_api::cloud_devices_links_prepare(
+				&openapi_config_cloud_unauthed,
+			)
+			.await
+		);
 
-		Ok(())
+		Ok(SideKickResponse(json!({
+			"device_link_url": prepare_res.device_link_url,
+			"device_link_token": prepare_res.device_link_token,
+		})))
```

Comment:
@NathanFlurry with regards to the previous method I used for error handling, how does this look?

@jog1t this should make it a bit cleaner for us to use in Sidekick :)

Reply:
In my head, I want a type layer there (even just as a marker) that acts as a barrier between what is happening in the CLI and what gets directly printed out. Otherwise, I feel like Sidekick will be super unstructured from the start. Returning a `GlobalResult<Value>` seems pretty bare to me.

Also, down the line, we can maybe look at making the type more specific, and not just a json value at the top. Say if we come up with a better schema on exactly everything that Sidekick could respond with.

That said, it's more of a feeling than anything, so I wouldn't be opposed to removing it.

Reply:
sgtm. would like to switch over to typed responses at some point. (can this be adopted incrementally?)